src/main/resources/mixins.common.realtime.json

```diff
@@ -0,0 +1,28 @@
+{
+  "minVersion": "0.5.1",
+  "package": "org.spongepowered.common.mixin.realtime.mixin",
```

Comment:
@Zidane: What?? I just meant that instead of "org.spongepowered.common.mixin.realtime.mixin", the package could just be "org.spongepowered.common.mixin.realtime", to match our other mixins (e.g. "org.spongepowered.mod.mixin.entityactivation").